src/sentry/runner/commands/cleanup.py

```diff
@@ -132,16 +129,21 @@ def multiprocess_worker(task_queue: _WorkQueue) -> None:
                 )
 
                 while True:
+                    debug_output(f"Processing chunk of {len(chunk)} {model_name} objects")
+                    metrics.incr(
+                        "cleanup.chunk_processed", tags={"model": model_name}, amount=len(chunk)
+                    )
                     if not task.chunk(apply_filter=True):
                         break
         except Exception:
             metrics.incr(
                 "cleanup.error",
                 instance=model_name,
```

Comment:
In this file, `instance` can represent anything, thus, it makes Datadog filtering hard.
Instead I'm going to use `model` and `db_router` across the various metrics and one day in the future drop the usage of `instance` (I want some overlap of the old and new formats).